data/sound-strings/ar.json/localize.json

```diff
@@ -1,107 +1,107 @@
-"onto":"إلى",
-"take_exit_number":"خروج إلى",
-"take_exit_number_street_verb":"NULL",
-"destination":"سوف تصل.",
-"you_have_reached_the_destination":"لقد وصلت.",
-"in_50_meters":"بعد خمسين متراً",
-"in_100_meters":"بعد مئة متر",
-"in_200_meters":"بعد مئتين متر",
-"in_250_meters":"بعد مئتين وخمسين متراً",
-"in_300_meters":"بعد ثلاثمئة متر",
-"in_400_meters":"بعد أربعمئة متر",
-"in_500_meters":"بعد خمسمئة متر",
-"in_600_meters":"بعد ستمئة متر",
-"in_700_meters":"بعد سبعمئة متر",
-"in_750_meters":"بعد سبعمئة وخمسين مترًا",
-"in_800_meters":"بعد ثمانمئة متر",
-"in_900_meters":"بعد تسعمئة متر",
-"in_1_kilometer":"بعد كيلومتر واحد",
-"in_1_5_kilometers":"بعد كيلومتر ونصف",
-"in_2_kilometers":"بعد كيلومترين",
-"in_2_5_kilometers":"بعد كيلومترين ونصف",
-"in_3_kilometers":"بعد ثلاثة كيلومترات",
-"then":"ثم",
-"dist_direction_onto_street":"%1$s %2$s %3$s %4$s",
-"take_the_1_exit":"اسلك المخرج الأول.",
-"take_the_1_exit_street":"NULL",
-"take_the_1_exit_street_verb":"NULL",
-"take_the_2_exit":"اسلك المخرج الثاني.",
-"take_the_2_exit_street":"NULL",
-"take_the_2_exit_street_verb":"NULL",
-"take_the_3_exit":"اسلك المخرج الثالث.",
-"take_the_3_exit_street":"NULL",
-"take_the_3_exit_street_verb":"NULL",
-"take_the_4_exit":"اسلك المخرج الرابع.",
-"take_the_4_exit_street":"NULL",
-"take_the_4_exit_street_verb":"NULL",
-"take_the_5_exit":"اسلك المخرج الخامس.",
-"take_the_5_exit_street":"NULL",
-"take_the_5_exit_street_verb":"NULL",
-"take_the_6_exit":"اسلك المخرج السادس.",
-"take_the_6_exit_street":"NULL",
-"take_the_6_exit_street_verb":"NULL",
-"take_the_7_exit":"اسلك المخرج السابع.",
-"take_the_7_exit_street":"NULL",
-"take_the_7_exit_street_verb":"NULL",
-"take_the_8_exit":"اسلك المخرج الثامن.",
-"take_the_8_exit_street":"NULL",
-"take_the_8_exit_street_verb":"NULL",
-"take_the_9_exit":"اسلك المخرج التاسع.",
-"take_the_9_exit_street":"NULL",
-"take_the_9_exit_street_verb":"NULL",
-"take_the_10_exit":"اسلك المخرج العاشر.",
-"take_the_10_exit_street":"NULL",
-"take_the_10_exit_street_verb":"NULL",
-"take_the_11_exit":"اسلك المَخْرَج الحادي عشر.",
-"take_the_11_exit_street":"NULL",
-"take_the_11_exit_street_verb":"NULL",
-"in_50_feet":"بعد خمسين قدمًا",
-"in_100_feet":"بعد مئة قدم",
-"in_200_feet":"بعد مئتين قدم",
-"in_300_feet":"بعد ثلاثمئة قدم",
-"in_400_feet":"بعد أربعمئة قدم",
-"in_500_feet":"بعد خمسمئة قدم",
-"in_600_feet":"بعد ستمئة قدم",
-"in_700_feet":"بعد سبعمئة قدم",
-"in_800_feet":"بعد ثمانمئة قدم",
-"in_900_feet":"بعد تسعمئة قدم",
-"in_1000_feet":"بعد ألف قدم",
-"in_1500_feet":"بعد ألف وخمسمئة قدم",
-"in_2000_feet":"بعد ألفي قدم",
-"in_2500_feet":"بعد ألفين وخمسمئة قدم",
-"in_3000_feet":"بعد ثلاثة آلاف قدم",
-"in_3500_feet":"بعد ثلاثة آلاف وخمسمئة قدم",
-"in_4000_feet":"بعد أربعة آلاف قدم",
-"in_4500_feet":"بعد أربعة آلاف وخمسمئة قدم",
-"in_5000_feet":"بعد خمسة آلاف قدم",
-"in_1_mile":"بعد ميل واحد",
-"in_1_5_miles":"بعد ميل ونصف",
-"in_2_miles":"بعد ميلين",
-"unknown_camera":"الكاميرا أمامك"
+    "make_a_slight_right_turn": "الزم اليمين.",
```

Comment:
Massive whitespace changes will cause merge conflicts with any outstanding translations. We should get the attention of any/all translation teams and contributors and ask them to finish any changes/PRs strategically (before this merge? save a diff and consider re-pulling after the merge? just be aware?) and also instruct them on the new processes for contributing translations. (Update our docs/*.md?) Finally, we should make developers very aware of the new processes for adding strings to the app and getting translations for those new strings ready for PR/beta/release.

Happy to collaborate on this, I'm just not central enough to know how to reach everyone.

Reply:
Let's temporary exclude sound-strings from this diff. I will disable the component temporary.